core/common/src/main/java/alluxio/grpc/GrpcManagedChannelPool.java

```diff
@@ -53,6 +54,17 @@
             .getMs(PropertyKey.MASTER_GRPC_CHANNEL_SHUTDOWN_TIMEOUT));
   }
 
+  /**
+   * Creates a new channel pool with the given health check timeout and channel shutdown timeout
+   *
+   * @param networkHealthCheckTimeoutMs timeout in ms for channel health check
+   * @param channelShutdownTimeoutMs timeout in ms for channel shutdown.
+   */
+  @VisibleForTesting
+  public static void renewChannelPool(long networkHealthCheckTimeoutMs, long channelShutdownTimeoutMs) {
+    sInstance = new GrpcManagedChannelPool(networkHealthCheckTimeoutMs, channelShutdownTimeoutMs);
+  }
```

Comment:
Do we need to clean up the old channel pool?

Reply:
I haven't seen any issues when _not_ cleaning it up. Because it's only used in for a select few testing scenarios I'm not sure if it matters too much. In the next few days I'll probably remove this method anyways because we'll likely be removing the singleton instance. This is more of a temporary fix.